api/scripts/clean-anonymized-users-authentication-methods.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+require('dotenv').config();
+
+const _ = require('lodash');
+const bluebird = require('bluebird');
+
+const { parseCsvWithHeader } = require('./helpers/csvHelpers');
+const authenticationMethodRepository = require('../lib/infrastructure/repositories/authentication-method-repository');
+
+async function cleanAnonymizedAuthenticationMethods({ arrayOfAnonymizedUsersIds }) {
+  const anonymizedUserIdsWithAuthenticationMethodsDeleted = [];
+  await bluebird.mapSeries(arrayOfAnonymizedUsersIds, async (userId) => {
+    const numberOfRowDeleted = await authenticationMethodRepository.removeAllAuthenticationMethodsByUserId({
+      userId,
+    });
+    if (numberOfRowDeleted > 0) {
+      anonymizedUserIdsWithAuthenticationMethodsDeleted.push(userId);
+    }
+  });
+  return anonymizedUserIdsWithAuthenticationMethodsDeleted;
+}
+
+async function main() {
+  console.log('Starting deleting anonymized users authentication methods');
+
+  try {
+    const filePath = process.argv[2];
+
+    const anonymizedUsersIds = await parseCsvWithHeader(filePath);
+    const arrayOfAnonymizedUsersIds = _(anonymizedUsersIds)
+      .map((user) => user.ID)
+      .filter((userId) => !_.isNil(userId))
+      .value();
```

Comment:
pourquoi pas le faire en pure js?
```suggestion
    const arrayOfAnonymizedUsersIds = anonymizedUsersIds
      .filter((user) => !_.isNil(user.ID));
      .map((user) => user.ID)
```

Reply:
y'a un petit avantage je crois, c'est que le filter de lodash ne throw pas sur les valeurs `undefined` , lodash apporte une petite surcouche de magie qui protège des cas un peu tordu du js natif